### PR TITLE
Update SOAP project creation task

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/apikit-4-soap-project-task.adoc [hunk 1]
```diff
@@ -1,27 +1,33 @@
-= To Create an APIkit for SOAP Project
+= Create an APIkit for SOAP Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You create a new project, specify the use of APIkit components, and refer to a WSDL file that you download.
+This process walks you through the creation of a create a new Mule project based on a SOAP API specification.
```
Comment (dhirsch1) [inline modules/ROOT/pages/apikit-4-soap-project-task.adoc]: Remove "a create."

File: modules/ROOT/pages/apikit-4-soap-project-task.adoc [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
 +
-The New Mule Project wizard appears.
-. In Project Name, enter a name for the project, for example *myproject*. Select or accept the default EE Runtime, for example Mule Server 4.1.0 EE.
-. In API Implementation, check Specify API Definition File Location or URL, and browse to the WSDL file you downloaded earlier.
+image::create-new-mule-project-soap.png[]
+<1> In *Project Name*, enter a name for the project. +
+For example _soap-implementation_.
+<1> Select the Mule runtime engine version. +
+For example `Mule Server 4.2.2 EE`.
+<1> Under *API Specification*, select the  *From Design Center or Local Files* tab.
```
Comment (dhirsch1) [inline modules/ROOT/pages/apikit-4-soap-project-task.adoc:17]: It looks like you have an extra space after "the."

File: modules/ROOT/pages/apikit-4-soap-project-task.adoc [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
+<1> Select the expand icon: and select *Browse files...*.
```
Comment (dhirsch1) [inline modules/ROOT/pages/apikit-4-soap-project-task.adoc]: The colon should not be there. I suggest: "Select the expand icon, and then select..."

File: modules/ROOT/pages/apikit-4-soap-project-task.adoc [hunk 4]
```diff
@@ -1,27 +1,33 @@
...
+= Create an APIkit for SOAP Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You create a new project, specify the use of APIkit components, and refer to a WSDL file that you download.
+This process walks you through the creation of a create a new Mule project based on a SOAP API specification.
 
-. Under the File menu, select New > Mule Project.
+Before proceeding, ensure that you meet the xref:apikit-4-soap-prerequisites-task.adoc[Prerequisites for Using APIkit for SOAP]
```
Comment (dhirsch1) [inline modules/ROOT/pages/apikit-4-soap-project-task.adoc]: Add a period to this sentence.

File: modules/ROOT/pages/apikit-4-soap-project-task.adoc [hunk 5]
```diff
@@ -1,27 +1,33 @@
...
+
+. Under the File menu, select *New* > *Mule Project*.
 +
-The New Mule Project wizard appears.
-. In Project Name, enter a name for the project, for example *myproject*. Select or accept the default EE Runtime, for example Mule Server 4.1.0 EE.
-. In API Implementation, check Specify API Definition File Location or URL, and browse to the WSDL file you downloaded earlier.
+image::create-new-mule-project-soap.png[]
+<1> In *Project Name*, enter a name for the project. +
+For example _soap-implementation_.
```
Comment (dhirsch1) [inline modules/ROOT/pages/apikit-4-soap-project-task.adoc]: Add a comma after "For example" here and elsewhere.

**Review by dhirsch1 — APPROVED** [review]
Approved with some things to correct

File: modules/ROOT/pages/apikit-4-soap-project-task.adoc (hunk 3 above)
Comment (fermujica) [reply modules/ROOT/pages/apikit-4-soap-project-task.adoc]: Thanks! I missed the image of the icon there!

**Pull request merged**